doc/whats_new/_contributors.rst

```diff
@@ -141,3 +141,5 @@
 .. _Neeraj Gangwar: http://neerajgangwar.in
 
 .. _Arthur Mensch: https://amensch.fr
+
+.. _Christian Braune: https://github.com/christianbraune79
```

Comment:
This is unused and unnecessary. These says we try to only use this format for core devs...